ontology/yaml/resources/HVAC/entity_types/FAN.yaml

```diff
@@ -636,4 +636,18 @@ FAN_SS_VSC_EPC_EHHRC_EHHRC3X:
   uses:
   - heat_recovery_water_isolation_valve_command_1
   - heat_recovery_water_isolation_valve_command_2
-  - heat_recovery_water_isolation_valve_command_3
\ No newline at end of file
+  - heat_recovery_water_isolation_valve_command_3
+  
+FAN_SS_SFVCS_SFVSC_FDPM_STC_SRC_MIPCVM_EPM_SFRDM:
+  description: "Fan with supply static pressure, power, temperature and run monitoring."
+  implements:
+  - FAN
+  - SS
+  - SFVCS
```

Comment:
`SFVSC` and `SFVCS`? How can you have both?

Reply:
the above canonical has been deleted